tests/analysis/test_pmdarima_analyzer.py

```diff
@@ -172,10 +170,7 @@ def test_pmdarima_calculate_acf_minimal_args(data):
     ).calculate_acf(unbiased=False, nlags=90, qstat=False, fft=False, alpha=None)
     for payload in acf_data.values():
         assert {"acf"}.issubset(payload.keys())
-        assert [
-            key not in payload.keys()
-            for key in ["qstat", "pvalues", "confidence_intervals"]
-        ]
+        assert [key not in payload.keys() for key in ["qstat", "pvalues", "confidence_intervals"]]
```

Comment:
```suggestion
        assert any(key not in payload.keys() for key in ["qstat", "pvalues", "confidence_intervals"])
```

Is `any` (or `all`) missing in this line?

Reply:
yep. Fixed that and another test that does a list exclusion search